kubernetes-examples/topicencryption/base/kroxylicious-deployment.yaml

```diff
@@ -0,0 +1,47 @@
+#
+# Copyright Kroxylicious Authors.
+#
+# Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+#
+
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: kroxylicious-proxy
+  labels:
+    app: kroxylicious
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: kroxylicious
+  template:
+    metadata:
+      labels:
+        app: kroxylicious
+    spec:
+      containers:
+      - name: kroxylicious
+        image: quay.io/kroxylicious/kroxylicious-developer:0.4.0-SNAPSHOT
+        imagePullPolicy: Always
+        args: ["--config", "/opt/kroxylicious/config/config.yaml"]
```

Comment:
Seems like a sensible default path should we update the default (in a new PR)?

Reply:
I suppose we could update the Dockerfile to assume that path, but that would require users (Docker user or Kubernetes user) to mount the config file in the same place.  I'm not sure that's too helpful.